CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## TBD
 
+BREAKING CHANGES:
+ - [crypto] Remove xchacha20poly1305 in favor of hkdfchachapoly1305
```

Comment:
why are you adding it here? was it done before?

Reply:
I had never added it to the changelog previously. I can remove it though

Reply:
Please do